lib/ansible/modules/cloud/amazon/cloudformation.py

```diff
@@ -300,6 +311,9 @@ def create_changeset(module, stack_params, cfn):
         module.fail_json(msg="Either 'template' or 'template_url' is required.")
     if module.params['changeset_name'] is not None:
         stack_params['ChangeSetName'] = module.params['changeset_name']
+
+    stack_params.pop('ClientRequestToken', None)
```

Comment:
Can you add a comment here? eg "not passing the token on because _______".

Reply:
Prefer `del` over `pop` when not using the return value.

Reply:
There's no "soft" `del` action I'm aware of - this is implemented with the `, None` in case the CRT parameter is missing.